modules/Core/src/main/java/org/terasology/logic/health/HealthAuthoritySystem.java

```diff
@@ -257,24 +270,6 @@ public void onCrash(HorizontalCollisionEvent event, EntityRef entity) {
         }
     }
 
-    @ReceiveEvent
-    public void onCrash(HorizontalCollisionEvent event, EntityRef entity, CharacterSoundComponent characterSounds, HealthComponent healthComponent) {
-        Vector3f horizVelocity = new Vector3f(event.getVelocity());
-        horizVelocity.y = 0;
-        float velocity = horizVelocity.length();
-
-        if (velocity > healthComponent.horizontalDamageSpeedThreshold) {
-            if (characterSounds.lastSoundTime + CharacterSoundSystem.MIN_TIME < time.getGameTimeInMs()) {
-                StaticSound sound = random.nextItem(characterSounds.landingSounds);
-                if (sound != null) {
-                    entity.send(new PlaySoundEvent(sound, characterSounds.landingVolume));
-                    characterSounds.lastSoundTime = time.getGameTimeInMs();
-                    entity.saveComponent(characterSounds);
-                }
-            }
-        }
-    }
```

Comment:
Any reason as to why this event handler has been removed? Looks like it is doing something different than the other one.